daemon/health.go

```diff
@@ -325,3 +321,13 @@ func min(x, y int) int {
 	}
 	return y
 }
+
+func getShell(config containertypes.Config) []string {
+	if len(config.Shell) != 0 {
+		return config.Shell
+	}
+	if runtime.GOOS != "windows" {
```

Comment:
This is duplicated with `builder/dockerfile/builder_unix.go` (and its windows counter-part), we might want to extract that 👼